Library/Formula/phantomjs.rb

```diff
@@ -26,11 +14,14 @@ class Phantomjs < Formula
     sha256 "45091efed57f4de5f04810a874e050206ae587ac85e17892f570e0e7eb50b977" => :mountain_lion
   end
 
+  depends_on "openssl"
+
   def install
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
-      "--qt-config", "-openssl-linked"
+    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
```

Comment:
It may be worth investigating if we can use `-no-openssl -securetransport` but that can be in a future PR.